internal/scti/handlers.go

```diff
@@ -68,48 +68,48 @@ var (
 	lastSCTTimestamp *prometheus.GaugeVec     // origin => value
 	reqsCounter      *prometheus.CounterVec   // origin, op => value
 	rspsCounter      *prometheus.CounterVec   // origin, op, code => value
-	rspLatency       *prometheus.HistogramVec // origin, op, code => value
+	rspDuration      *prometheus.HistogramVec // origin, op, code => value
 )
 
 // setupMetrics initializes all the exported metrics.
 func setupMetrics() {
 	// TODO(phboneff): add metrics for deduplication and chain storage.
 	knownLogs = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "known_logs",
+			Name: "ct_known_logs_info",
 			Help: "Set to 1 for known logs",
 		},
 		[]string{"origin"})
 	lastSCTTimestamp = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "last_sct_timestamp",
-			Help: "Time of last SCT in ms since epoch",
+			Name: "ct_sct_last_timestamp_seconds",
+			Help: "Time of last SCT in seconds since epoch",
 		},
 		[]string{"origin"})
 	lastSCTIndex = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "last_sct_index",
+			Name: "ct_sct_last_index",
 			Help: "Index of last SCT",
 		},
 		[]string{"origin"})
 	reqsCounter = promauto.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "http_reqs",
+			Name: "ct_http_requests_total",
 			Help: "Number of requests",
 		},
-		[]string{"origin", "ep"})
+		[]string{"origin", "operation"})
 	rspsCounter = promauto.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "http_rsps",
+			Name: "ct_http_responses_total",
 			Help: "Number of responses",
 		},
-		[]string{"origin", "op", "code"})
-	rspLatency = promauto.NewHistogramVec(
+		[]string{"origin", "operation", "code"})
+	rspDuration = promauto.NewHistogramVec(
 		prometheus.HistogramOpts{
-			Name: "http_latency",
+			Name: "ct_http_request_duration_seconds",
```

Comment:
nit: variable naming is in terms of "response", but metric name is now in terms of "request" - would be good to keep them consistent if possible.

Reply:
Oh yeah, good point!